pincer/objects/guild/guild.py

```diff
@@ -349,7 +351,9 @@ class Guild(APIObject):
     system_channel_flags: APINullable[SystemChannelFlags] = MISSING
     explicit_content_filter: APINullable[ExplicitContentFilterLevel] = MISSING
     premium_tier: APINullable[PremiumTier] = MISSING
-    default_message_notifications: APINullable[DefaultMessageNotificationLevel] = MISSING
+    default_message_notifications: APINullable[
```

Comment:
I think it might be better for anyone to either shorter our names or increase the 80 char limit...

Reply:
> You probably noticed the peculiar default line length. Black defaults to 88 characters per line, which happens to be 10% over 80. This number was found to produce significantly shorter files than sticking with 80 (the most popular), or even 79 (used by the standard library). In general, 90-ish seems like the wise choice.

^From Black

I think we should increase it

Reply:
+1

Reply:
+1

Reply:
+1